### PR TITLE
[Enhance] Clarity for Kitty Dcol

### PR DESCRIPTION
**Pull request by kRHYME7**
Commented out some color override as these are critical on terminal
 
Goal of this change is to give clarity. 

Based of zsh
+ The prompt uses colors [ 7 , 4 , 0  ]
image
 
+ The commands and executables is color [ 2 ]
image

+ Here's some overrides for the color.
image

WARNINGS, ERRORS, ALERTS are not visible if we override these colors.
Visibility in Terminal is vital, therefore I commented out these colors. 
 
```
color1 
color3 
color5 
color6 
color8 
color9 
color10
color11
color12
color13
color14
color15
```

##### Also tries to tweak for fish compatibility.  

image

My bad for this frequent change.

### CODE REVIEW

**Comment by kRHYME7** [comment]
I'll be adding toggle for presets on ``` hyprdots-ctl ``` also a switch to turn off wallbash for certain package.

**Comment by prasanthrangan** [comment]
cool thanks, but it now looks more like default colors...
how does the pry0 and txt0 color look like? it should have enough contrast to make it bg and fg.

**Comment by kRHYME7** [comment]
> how does the pry0 and txt0 color look like? it should have enough contrast to make it bg and fg.

How about using this instead as default?  This is my first attempt but too sharp to be a terminal 
```
foreground         #<wallbash_txt0>
background         #<wallbash_pry0>
``` 


So decided to use this softer approach, but I think I can add this as a preset, so let's use the above config?

```
foreground         #<wallbash_pry2>
background         #<wallbash_0xa1>
```

File: Configs/.config/hypr/wallbash/kitty.dcol [hunk 1]
```diff
@@ -1,7 +1,6 @@
 $HOME/.config/kitty/themes/Wall-Dcol.conf|${ScrDir}/wallbashkitty.sh
-foreground         #<wallbash_pry2>
-#background        #<wallbash_pry0>  # This looks way better but not good for low contrast, high brightness walls
-background         #<wallbash_0xa1>
+foreground         #<wallbash_txt0>
+background         #<wallbash_txt0>
```
Comment (prasanthrangan) [inline Configs/.config/hypr/wallbash/kitty.dcol:3]: looks like typo here, bg should be pry0

**Comment by kRHYME7** [comment]
@prasanthrangan  Thanks! ``` Hyprdots wallbash term -t ``` is out for testing.

**Review by prasanthrangan — APPROVED** [review]
cool, looks good now!

**Pull request merged**